das-function.yml

```diff
@@ -5,7 +5,7 @@ provider:
 functions:
   query-engine:
     lang: python3-http
-    image: trueagi/das:v1.5.0-queryengine
+    image: trueagi/das:v1.6.0-queryengine
```

Comment:
Approved but I'd like to understand what is this versioning and when it'd need to be updated?